corehq/blobs/tests/test_export.py

```diff
@@ -1,123 +1,90 @@
 import doctest
-import os
 import tarfile
-import uuid
 from io import BytesIO, RawIOBase
 from tempfile import NamedTemporaryFile
 
 from django.test import SimpleTestCase, TestCase
 
-from corehq.apps.hqmedia.models import (
-    CommCareAudio,
-    CommCareImage,
-    CommCareVideo,
-)
 from corehq.blobs import CODES, get_blob_db
-from corehq.blobs.export import EXPORTERS
+from corehq.blobs.export import BlobExporter
+from corehq.blobs.management.commands.run_blob_import import Command as ImportCommand
+from corehq.blobs.tests.fixtures import blob_db
 from corehq.blobs.tests.util import TemporaryFilesystemBlobDB, new_meta
 
 
-class TestBlobExport(TestCase):
-    @classmethod
-    def setUpClass(cls):
-        super().setUpClass()
+class TestBlobExporter(TestCase):
```

Comment:
Was this class completely rewritten? As in the old tests were removed and new/different ones were added? I'm finding it difficult to understand what was changed. It might have been worth splitting https://github.com/dimagi/commcare-hq/pull/35343/commits/e6e825c42f6f5d057177afc045feb466394f26cb into multiple commits: one that removes the old tests and one that adds new ones.

Reply:
Yeah it didn't occur to me at the time of rewriting the test, but can understand how the review is hard to reason about. Sorry about that!